e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -309,20 +325,133 @@ describe('Upload Files', () => {
         cy.postMessage('test');
         cy.findByTestId('fileAttachmentList').find('.post-image').should('have.length', 5);
     });
-});
```

Comment:
With the changes on files, L233 failed on me when running all tests in this spec. It should now be:
```diff
- const attachmentFilename = 'image-file.jpg';
+ const attachmentFilename = 'jpg-image-file.jpg';
```

Please run all the tests and let me know if all are consistently passing on you.